nautobot/extras/urls.py

```diff
@@ -170,6 +170,21 @@
         kwargs={"model": ConfigContextSchema},
     ),
     # contacts
+    path(
+        "contact-associations/map-contact-from-location",
+        views.MapContactorTeamFromLocationView.as_view(),
+        name="map_contact_from_location",
+    ),
+    path(
+        "contact-associations/map-new-contact-from-location",
+        views.MapNewContactFromLocationView.as_view(),
+        name="map_new_contact_from_location",
+    ),
+    path(
+        "contact-associations/map-new-team-from-location",
```

Comment:
missing trailing slashes on these paths

```suggestion
        "contact-associations/map-contact-from-location/",
        views.MapContactOrTeamFromLocationView.as_view(),
        name="map_contact_from_location",
    ),
    path(
        "contact-associations/map-new-contact-from-location/",
        views.MapNewContactFromLocationView.as_view(),
        name="map_new_contact_from_location",
    ),
    path(
        "contact-associations/map-new-team-from-location/",
```

Reply:
I wonder though, should these patterns also include a `<uuid: location_pk>` parameter?